addons/calendar/static/src/xml/base_calendar.xml

```diff
@@ -41,6 +41,12 @@
                 </div>
             </div>
         </t>
+        <t t-jquery=".o_cw_popover_fields_secondary, .o_cw_popover_edit" t-operation="attributes">
+            <attribute name="t-if">widget.isEventPrivate() ? widget.isCurrentPartnerAttendee() : true</attribute>
+        </t>
+        <t t-jquery=".o_cw_popover_delete" t-operation="attributes">
+            <attribute name="t-if">widget.canDelete and (widget.isEventPrivate() ? widget.isCurrentPartnerAttendee() : true)</attribute>
+        </t>
```

Comment:
Don't really like that we have to _override_ `t-if` by _overwrite_.
I think we can avoid this by having base template `t-if`s on each item which points to JS public methods.
That way, we can cleanly override without duplicate code.
Example:

```xml
<!-- (web) template -->
<a t-if="widget.isEventDeleteable()" class="o_cw_popover_delete">Delete</a>
```
```js
// (web) calendar popover
isEventDeletable() {
   return this.canDelete; // Note: can turn this attribute private, so prefix with `_`
},

// (calendar) calendar popover
isEventDeletable() {
   return (
      this._super() &&
      (
         this.isEventPrivate() // Note: could become private
            ? this.isCurrentPartnerAttendee()
            : true
      )
   );
},
```